react/components/SliderTrack.tsx

```diff
@@ -74,7 +88,7 @@ const SliderTrack: FC<{ totalItems: number }> = ({ children, totalItems }) => {
         return (
           <div
             key={index}
-            className={`flex relative ${handles.slide}`}
+            className={`flex relative ${handles.slide} ${getFirstOrLastVisible(handles, slidesPerPage, index)}`}
```

Comment:
I think is better to use the `applyModifiers`, [here is an example](https://github.com/vtex-apps/slider-layout/blob/1e680477a25cd434811c10d036791793aa1881a2/react/components/PaginationDots.tsx#L86)